test/unit/domain/stateMachine/verifyTransaction.model.test.ts

```diff
@@ -25,6 +25,17 @@ optionally within square brackets <email>.
 --------------
 ******/
 
+/* IMPORTANT
+   The fido challenges found in `auth-service` are signed with
+   Kevin Leyow's <kevin.leyow@modusbox.com> Yubikey. If the POST /consent
+   `consentId` and `scopes` ever change form you will need to derivie and resign the challenges,
+   update the `credential` object and update this PSA.
+   You will also need to update the public keys found in every verify transaction flow.
+   Use https://github.com/mojaloop/contrib-fido-test-ui to retrieve data used to update
+   the response bodies.
+*/
```

Comment:
Maybe add a link to the steps for that from here: https://github.com/mojaloop/contrib-fido-test-ui

Reply:
done